docs/guides/lint-arazzo.md

```diff
@@ -120,6 +120,16 @@ To expand the linting checks for an Arazzo description, start by enabling
 some of the built-in rules. The currently-supported rules are:
 
 - `parameters-no-body-inside-in`: the `in` section inside `parameters` must not contain a `body`.
+- `strict-source-description-type`: the `type` property of the `sourceDescription` object must be either `openapi` or `arazzo`.
+- `arazzo-version-enum`: the `version` property must be one of the allowed values: [`1.0.0`].
+- `workflow-workflowId-unique`: the `workflowId` property must be unique across all workflows.
+- `step-stepId-unique`: the `stepId` must be unique amongst all steps described in the workflow.
+- `sourceDescription-name-unique`: the `name` property of the `sourceDescription` object must be unique across all source descriptions.
+- `workflow-dependsOn-unique`: the items in the `workflow` `dependsOn` property must be unique.
+- `parameters-unique`: the `parameters` list must not include duplicate parameters.
```

Comment:
I'm thinking about prepending all the arazzo-specific rules with `arazzo-`, e.g.: 

```suggestion
- `arazzo-parameters-unique`: the `parameters` list must not include duplicate parameters.
```

This way we make it clear that the rule is specific to the Arazzo specification and not to the parameters inside OAS/AsyncAPI. 

Also, we can prefix the Spot-related rules with `spot-`.

Reply:
For me it will looks like smth extra and not required, as the top section already describes `ArazzoRules`:

```
arazzoRules:
  spec: error
  arazzo-parameters-no-body-inside-in: error
  arazzo-strict-source-description-type: error
```
@lornajane , what do you think ?

Reply:
The rules either have same-named equivalents for the other formats (AsyncAPI doesn't seem to have rules yet but probably should), or they don't exist for this format. What happens if you enable an OpenAPI 3 only rule for an OpenAPI 2 description? We might already handle this situation nicely.

Reply:
This makes sense, but there's nothing in the docs to explain that it's not just under `rules:`, so I would definitely have done this wrong. I'll push a docs update quickly for this PR.

Reply:
oh, we should update this in the top-level config docs as well ... I'll put a note in the followup docs ticket